kubemarine/resources/configurations/compatibility/kubernetes_versions.yaml

```diff
@@ -137,12 +145,30 @@ compatibility_map:
     kubernetes-dashboard: v2.7.0
     local-path-provisioner: v0.0.26
     crictl: v1.29.0
+  v1.28.9:
+    calico: v3.28.0
+    nginx-ingress-controller: v1.10.0
+    kubernetes-dashboard: v2.7.0
+    local-path-provisioner: v0.0.26
+    crictl: v1.30.0
   v1.29.1:
     calico: v3.27.0
     nginx-ingress-controller: v1.9.6
     kubernetes-dashboard: v2.7.0
     local-path-provisioner: v0.0.26
     crictl: v1.29.0
+  v1.29.4:
+    calico: v3.28.0
+    nginx-ingress-controller: v1.10.0
+    kubernetes-dashboard: v2.7.0
+    local-path-provisioner: v0.0.26
+    crictl: v1.30.0
+  v1.30.1:
```

Comment:
To support v1.30.x, it is necessary to include code changes and respect comments from #647

Reply:
removed v1.30.x from this PR